katcp/resource.py

```diff
@@ -595,10 +595,15 @@ def parent_name(self):
 
     @property
     def name(self):
-        """Name of this KATCP resource"""
+        """Name of this KATCPSensor"""
         return self._name
 
     @property
+    def python_id(self):
```

Comment:
@martinslabber also said that for katstore query they always normalise the name they get so it does not matter what the client provides ... we will probably do the same thing in katportal. I should create a doodle ... ha.